homeassistant/components/ultrasync/__init__.py

```diff
@@ -0,0 +1,170 @@
+"""The Interlogix UltraSync Hub component."""
+
+import asyncio
+
+from ultrasync import AlarmScene
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PIN,
+    CONF_SCAN_INTERVAL,
+    CONF_USERNAME,
+)
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    DATA_COORDINATOR,
+    DATA_UNDO_UPDATE_LISTENER,
+    DEFAULT_NAME,
+    DEFAULT_SCAN_INTERVAL,
+    DOMAIN,
+    SERVICE_AWAY,
+    SERVICE_DISARM,
+    SERVICE_STAY,
+)
+from .coordinator import UltraSyncDataUpdateCoordinator
+
+PLATFORMS = ["sensor"]
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
Where is this used?

Reply:
I'm just following what appears to be a standard in the `homeassistant/components/` directory.  Is it not needed?
```bash
grep CONFIG_SCHEMA homeassistant/components/ -Rl | egrep -v __pycache | wc -l
# outputs: 343
```
I checked in some of those entries and all of ones i chose (big names such as `zwave`, `wink`, `tesla`, `tplink`, etc ) didn't use it either; i figured it was some sort of standard to have it?

Reply:
It's linked to YAML validation. 
No YAML = no CONFIG_SCHEMA